### DISCUSSION TITLE
AdGuard Home LXC - Install error validating ports: listen tcp 0.0.0.0:53: bind: address already in use

### DISCUSSION THREAD

**Discussion by gamersalpha** [tteck/Proxmox#987] [Help Needed] [answered]
Hello i want try and use for production AdGuard Home LXC 

but i got an issue : 

![image](https://user-images.githubusercontent.com/32466950/214899396-5501ec42-16b9-4609-976a-358c6c0213c3.png)

so he is a frech install with the link : 

`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/adguard-v5.sh)"`

do u know how fix the problem with Proxmox and AdGuard Home LXC  installation

thnaks

**Comment by tteck** [comment, marked as answer]
You can see whats using port 53 by running `lsof -i :53`

To fix the issue, run the commands below in the AdGuard LXC console
```
echo "DNSStubListener=no" >>/etc/systemd/resolved.conf
ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
```
reboot the LXC
You can again run `lsof -i :53` to ensure port 53 is free

I think that I will edit the script to free port 53 before installing AdGuard. So, you can fix your installation, or wait for the commit.

**Reply by tteck** [reply]
Script has been fixed

**Comment by gamersalpha** [comment]
thanks u all, that's work !!!
